### DISCUSSION TITLE
Welcome to adblocker Discussions!

### DISCUSSION THREAD

**Discussion by remusao** [ghostery/adblocker#1498] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Olek97** [comment]
Hello, how can I implement this adblocker on my site? I put a live stream on my site that has ads and popups.

**Reply by remusao** [reply]
Hi @Z-Olek,

Can you be a bit more specific about what you'd like to achieve. Do you have a stream embedded on your site using an iframe or video element?

**Reply by Olek97** [reply]
Iframe

**Reply by remusao** [reply]
I do not think a web page has the required capabilities to intercept network requests and do the blocking, this would have to be done from an extension/browser.
